src/test/TestHelper.h

```diff
@@ -75,10 +76,13 @@ class AESDecrypter : public IAESDecrypter
 class DASHTestTree : public adaptive::DASHTree
 {
 public:
-  uint64_t mock_time = 10000000L;
-  std::chrono::system_clock::time_point mock_time_chrono = std::chrono::system_clock::now();
-  DASHTestTree();
-  uint64_t GetNowTime() override { return mock_time; }
-  std::chrono::system_clock::time_point GetNowTimeChrono() { return mock_time_chrono; };
+  DASHTestTree(UTILS::PROPERTIES::KodiProperties kodiProps);
+  uint64_t GetNowTime() override { return m_mock_time; }
+  void SetNowTime(uint64_t time) { m_mock_time = time; }
   void SetLastUpdated(std::chrono::system_clock::time_point tm) override { lastUpdated_ = tm; };
+  std::chrono::system_clock::time_point GetNowTimeChrono() { return m_mock_time_chrono; };
+
+private:
+  uint64_t m_mock_time = 10000000L;
```

Comment:
no big deal but should we be dropping the underscores in the name i.e `m_mockTime` ?

Reply:
yes my intention but also my oversight :-D

Reply:
All good! I'm not too worried about getting everything perfect first time. We can always do a couple of cleanups later on to get things that were missed beforehand.